components/PathContent.jsx

```diff
@@ -8,10 +8,29 @@ const StyledLink = styled.a`
   color: ${COLOR.GREY_2};
 `;
 
+const SvmDeposit = () => (
+  <>
+    Deposit SOL or WSOL and OLAS tokens using the Lockbox via
+    {' '}
+    <StyledLink href="https://tokenomics.olas.network/manage-solana-products" target="_blank" rel="noopener noreferrer">
```

Comment:
is this link correct? I see 404

Reply:
yeah its in: https://github.com/valory-xyz/autonolas-tokenomics-frontend/pull/116